apache/airflow
# Make skip of trigger form in UI if no params are defined configurable

This PR is a hotfix for Airflow 2.7.0rc1 adding a configuration option so that the trigger form on UI can be displayed also for DAGs w/o `params` defined which was removed in #31583.

How to test?
- Start Airflow Webserver on this PR
- Check that form still displays (always) when triggering DAG "example_bash_operator" - as this has a param defined
- Trigger the example_python_operator and see it starts w/o displaying UI
- Stop airflow webserver and start it with `AIRFLOW__WEBSERVER__SKIP_TRIGGER_FORM_IF_NO_PARAM=False airflow webserver -d` and trigger the example_python_operator - See it displays the JSON entry form (again)

![Screenshot 2023-08-12 at 18-59-47 Airflow](https://github.com/apache/airflow/assets/95105677/46afbe66-faf1-45e8-87c2-5bb6b7427d79)

closes: #33344
